src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,20 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+class LetterRemover private constructor() {
+    companion object {
+        @JvmStatic
+        val letterReplaceWith = mapOf(Pair('r', '!'))
+
+        @JvmStatic
+        fun removeLetter(input: String): CharArray {
```

Comment:
There is 'in-box' solution
Please check this link
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html

Reply:
Yes, that is an interesting function and its ability to ignore case. Thanks.